CHANGELOG.md

```diff
@@ -2,6 +2,76 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
+## [1.64.0](https://github.com/aws/aws-cdk/compare/v1.63.0...v1.64.0) (2020-09-22)
+
+
+### ⚠ BREAKING CHANGES TO EXPERIMENTAL FEATURES
+
+* **codedeploy:** the default policy for `LambdaDeploymentGroup` no longer contains `sns:Publish` on `*` permissions
+* **cfn-include:** the construction property 'nestedStacks' of class 'CfnInclude' has been renamed to 'loadNestedStacks'
+* **rds:** removed protected member `subnetGroup` from DatabaseCluster classes
+* **rds:** Cluster now has deletionProtection enabled if its removal policy is `RETAIN`
+* **rds**: Instance now has deletionProtection enabled by default only if its removal policy is `RETAIN`
```

Comment:
Duplicate

Reply:
Don't believe so - one is for `Instance` and the other is for `Cluster`

Reply:
Correct, and the behavior is _every-so-slightly-and-subtly-different_ between the two. Cluster used to not default deletionProtection (so always false by default), but now enables it if RETAIN is the removal policy. Instance used to always default to deletion protection, but now doesn't default to true if the removal policy is SNAPSHOT/DELETE (i.e., != RETAIN).